documentation/src/main/markdown/packagemgrs/golang.md

```diff
@@ -9,14 +9,15 @@
 [detect_product_short] has four detectors for GoLang:
 
 * Go Mod CLI (GO_MOD) detector (recommended)
+* Go Mod File (GO_MOD) detector
 * Go Lock (GO_DEP) detector
 * Go Gradle (GO_GRADLE) detector
 * Go Vendor (GO_VENDOR) detector
 * Go Vndr (GO_VNDR) detector
 
 ## Go Mod CLI (GO_MOD) detector
 
-* Discovers dependencies of go language (GoLang) projects.
+* Discovers dependencies of Go projects that uses Go Mod package manager.
```

Comment:
Perhaps a small change to reflect the go mod command versus the Go Module package manager?
* Discovers dependencies of Go projects that use Go Module package manager.